tools/documentation/assets/js/Search.js

```diff
@@ -122,14 +122,19 @@ function loadJSON(url, callback) {
   req.send();
 }
 
+function isComponentIndex() {
+  return /^\/docs\/[\w-]+\/$/.test(location.pathname);
+}
 Search.prototype.loadStore = function() {
-  loadJSON('./store.json', function(err, object) {
+  var storePath = isComponentIndex() ? '../store.json' : 'store.json';
```

Comment:
@Rajdeepc It's not a good practice to alter pathing that is environment-based using front-end JS. What we want to do is make sure it renders locally and set up the appropriate redirects in netlify if necessary.